relay-server/src/services/project.rs

```diff
@@ -862,9 +478,11 @@ impl Project {
             return;
         }
 
+        self.state = state;
+
         // Flush all waiting recipients.
         relay_log::debug!("project state {} updated", self.project_key);
-        channel.inner.send(state);
+        channel.inner.send(self.current_state());
```

Comment:
```suggestion
        channel.inner.send(state.current_state(&self.config));
        self.state = state;
```

This parses better in my head, doesn't make me check the implementation of `self.current_state()` and making sure I didn't actually forget to set something `self.current_state()` is depending on.